README.md

```diff
@@ -76,9 +76,10 @@ napari-growth-cone-finder/
 │       ├── __init__.py
 │       ├── test_dock_widget.py
 │       └── test_reader.py
+├── pyproject.toml
 ├── README.md
 ├── requirements.txt
-├── setup.py
+├── setup.cfg
```

Comment:
This is just my ignorance, but why do we still need both `pyprojet.toml` and `setup.cfg`?

Reply:
cause ya just do :)  it's the PEP 517 pattern
https://setuptools.pypa.io/en/latest/setuptools.html#setup-cfg-only-projects.